packages/api/src/command/medical/patient/create-consolidate-data.ts

```diff
@@ -89,3 +92,41 @@ const convertCollectionBundleToTransactionBundle = ({
 
   return transactionBundle;
 };
+
+const removeUnwantedFhirData = (data: Bundle): Bundle => {
+  return {
+    resourceType: data.resourceType,
+    id: data.id,
+    type: data.type,
+    entry: data.entry?.map(replaceCodingSystem),
+  };
+};
+
+const replaceCodingSystem = (resourceEntry: BundleEntry): BundleEntry => {
+  return {
+    response: {
+      ...resourceEntry.response,
+      ...(resourceEntry.response?.outcome
+        ? {
+            outcome: {
+              ...resourceEntry.response.outcome,
+              issue: resourceEntry.response.outcome.issue?.map(issue => {
+                return {
+                  ...issue,
+                  details: {
+                    ...issue.details,
+                    coding: issue.details?.coding?.map(coding => {
+                      return {
+                        ...coding,
+                        system: DATA_SOURCE_EXTENSION_URL,
```

Comment:
I recommend using another URL, just b/c the data source one defines possible data sources (Metriport, CommonWell, etc.), and here we're defining the URL of the system that defines the possible status /outcomes for FHIR operations, right?

Maybe something like `OPERATION_OUTCOME_EXTENSION_URL`?

Reply:
@leite08 Cool yea just updated this and the docs payload